sheet_to_triples/field.py

```diff
@@ -183,8 +188,13 @@ def must_not_exist(self):
 
     @property
     def exists(self):
-        # Note, a whitespace-only string does exist, may be confusing!
-        return bool(self._value)
+        if not self._value:
+            return False
+        # A whitespace-only string does indeed exist, so check we
+        # aren't dealing with one of those.
+        if isinstance(self._value, str) and self._value.isspace():
+            return False
```

Comment:
This comment is not clear. It still says a whitespace only string exists, but is returning `False`.